dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ItemCitationsRestRepository.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.repository;
+
+import java.sql.SQLException;
+import java.util.UUID;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.BadRequestException;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.app.rest.exception.RepositoryMethodNotImplementedException;
+import org.dspace.app.rest.model.ItemCitationsRest;
+import org.dspace.content.Item;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.dspace.uclouvain.citations.ItemCitation;
+import org.dspace.uclouvain.citations.UCLouvainCitationsService;
+import org.dspace.uclouvain.citations.UnknownCitationFormatException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Component;
+
+/**
+ * Rest repository to get a specific citation format for a given item.
+ * 
+ * @author Michaël Pourbaix (michael.pourbaix@uclouvain.be)
+ */
+@Component(ItemCitationsRest.CATEGORY + "." + ItemCitationsRest.PLURAL_NAME)
+public class ItemCitationsRestRepository extends DSpaceRestRepository<ItemCitationsRest, UUID> {
+
+    @Autowired
+    ItemService itemService;
+
+    @Autowired
+    UCLouvainCitationsService uclouvainCitationsService;
+
+    @Override
+    @PreAuthorize("hasPermission(#id, 'ITEM', 'READ')")
+    public ItemCitationsRest findOne(Context context, UUID id) {
+        HttpServletRequest request = getRequestService().getCurrentRequest().getHttpServletRequest();
+        String format = request.getParameter("format");
+        try {
+            Item item = itemService.find(context, id);
+            if (item == null) {
+                throw new ResourceNotFoundException("No such item: " + id);
+            }
+            ItemCitationsRest citationsRest = new ItemCitationsRest();
+            if (StringUtils.isEmpty(format)) {
```

Comment:
An idea :: is it possible to force "format" parameter required ? and check `if (format.equals("all"))` here ? To avoid a lot of server resource consuming if we call this URL without format ?